metals/src/main/scala/scala/meta/internal/metals/watcher/FileWatcherEvent.scala

```diff
@@ -22,4 +23,6 @@ object FileWatcherEvent {
     FileWatcherEvent(EventType.Modify, path)
   def delete(path: Path): FileWatcherEvent =
     FileWatcherEvent(EventType.Delete, path)
+  def overflow(path: Path): FileWatcherEvent =
+    FileWatcherEvent(EventType.Overflow, path)
```

Comment:
What exactly indicates `EventType.Overflow`? I think it'll be worth adding some comment next to the `case object Overflow extends EventType`

Reply:
Overflow indicates that the file watcher was overwhelmed and may have lost some events . I will add a comment